core/tests/webdriverio_utils/workflow.js

```diff
@@ -249,6 +249,18 @@ var addExplorationPlaytester = async function(username) {
   await _addExplorationRole('Playtester', username);
 };
 
+// Here, roleName is the server-side form of the name (e.g. 'owner').
+var _getExplorationRoles = async function(roleName) {
+  var ownerLocator = '.e2e-test-role-' + roleName + '-name';
```

Comment:
Can you please explain this line because it seems to be a bit different from the protractor code?

Reply:
As the page which is being tested here is still in Angular JS, protractor was using `by.repeater` but as there is no replacement for this in webdriverio, I just added a classname and used all selector for getting the text.

Reply:
Can we not have a more meaningful name here in terms of class?

Reply:
Done.

Reply:
But still `explorationOwnerLocator` name doesn't seem to be related to the roles part.

Reply:
Done.

Reply:
`explorationRoleNameLocator` would have been better

Reply:
Done.